react/features/conference/components/web/InviteMore.js

```diff
@@ -0,0 +1,93 @@
+// @flow
+import React from 'react';
+
+import { beginAddPeople } from '../../../invite';
+import { connect } from '../../../base/redux';
+import { getParticipantCount } from '../../../base/participants';
+import { Icon, IconInviteMore } from '../../../base/icons';
+import { isToolboxVisible } from '../../../toolbox';
+import { translate } from '../../../base/i18n';
+
+type Props = {
+
+    /**
+     * Whether tile view is enabled.
+     */
+    _tileViewEnabled: Boolean,
+
+    /**
+     * Whether to show the option to invite more people
+     * instead of the subject.
+     */
+    _visible: boolean,
+
+    /**
+     * Handler to open the invite dialog.
+     */
+    onClick: Function,
+
+    /**
+     * Invoked to obtain translated strings.
+     */
+    t: Function
+}
+
+/**
+ * Represents a replacement for the subject, prompting the
+ * sole participant to invite more participants.
+ *
+ * @param {Object} props - The props of the component.
+ * @returns {React$Element<any>}
+ */
+function InviteMore({
+    _tileViewEnabled,
+    _visible,
+    onClick,
+    t
+}: Props) {
+    return (
+        _visible
+            ? <div className = { `invite-more-container${_tileViewEnabled ? ' elevated' : ''}` }>
```

Comment:
is there a problem if we always apply the elevated class?

Reply:
Yeah, in large view the header will cover the (me) preview tile, due to its lower z-index.

Reply:
You mean the header covers the entire large view and not just a top portion of it?